### PR TITLE
[troubleshooting] Mention the potential requirement for full token service path

### PR DESCRIPTION
**Pull request by newtork**
## What Has Changed?

Customer noticed that the full service path may be necessary. I confirmed this assumption with our development setup. We've never encountered the exception before,

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/java/troubleshooting.mdx [hunk 1]
```diff
@@ -200,3 +200,18 @@ While generating a type-safe API client for an OData service, you encounter the
...
+:::info **Symptoms**
+When running `Destination#asHttp()` you encounter an exception message like `"DestinationAccessException: Failed to read authentication token. The destination service responded with an error: Could not verify the provided CSRF token because your session was not found"`.
+:::
+
+**Possible causes:**
+
+- The Destination Service on Cloud Foundry is unable to generate an authentication token that will be used in later requests.
+  In the service configuration, the destination entry references a token service endpoint that could not communicate correctly with the Destination Service.
+
+  To solve that issue, please check every field in the destination configuration for valid input.
```
Comment (artemkovalyov) [inline docs/java/troubleshooting.mdx:216]: Maybe a link to destination configuration or a short example of correct and incorrect one?

**Review by artemkovalyov — APPROVED** [review]
Looks good. A bit of extra context would be great.

File: docs/java/troubleshooting.mdx (hunk 1 above)
Comment (newtork) [reply docs/java/troubleshooting.mdx:216]: > A bit of extra context would be great.

(internal) https://github.wdf.sap.corp/MA/sdk/issues/5854 :)

> Maybe a link to destination configuration

Good idea!

**Comment by artemkovalyov** [comment]
@newtork, should we merge it?

**Pull request merged**